content/tutorial-build-run-rippled-ubuntu.md

```diff
@@ -63,23 +63,23 @@ These instructions use Ubuntu's APT (Advanced Packaging Tool) to install the sof
 
       a. Download Boost 1.65.0.
 
-          wget https://dl.bintray.com/boostorg/release/1.65.0/source/boost_1_65_0.tar.gz
+          wget https://dl.bintray.com/boostorg/release/1.65.0/source/boost_1_65_0.tar.gz ***TODO: If 1.66.0: https://dl.bintray.com/boostorg/release/1.66.0/source/boost_1_66_0.tar.gz***
 
-      b. Untar `boost_1_65_0.tar.gz`.
+      b. Untar `boost_1_65_0.tar.gz`. ***TODO: If 1.66.0, boost_1_66_0.tar.gz***
 
-          tar xvzf boost_1_65_0.tar.gz
+          tar xvzf boost_1_65_0.tar.gz ***TODO: If 1.66.0, boost_1_66_0.tar.gz***
 
-      c. In the new `boost_1_65_0` directory, run:
+      c. In the new `boost_1_65_0` directory, run: ***TODO: If 1.66.0, boost_1_66_0***
```

Comment:
Nit: When the instructions are this granular, it's best to provide examples of every command line you need to enter, even when they're super-basic things like `cd boost_1_66_0`. Otherwise, people skimming just the code examples will miss it.

Also, if you could explain what the bootstrap script does, that'd be nice. If not, no big deal.